src/main/java/org/folio/dew/domain/dto/ItemIdentifier.java

```diff
@@ -4,9 +4,18 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import static org.folio.dew.utils.Constants.UTF8_BOM;
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class ItemIdentifier {
   private String itemId;
+
+  public void setItemId(String itemId) {
```

Comment:
Does it make sense to rename method to void clearBomSymbol() without itemId specification because this method must only change state of ItemIdentifier dto?

Reply:
Here we need just to override setter for itemId because it may contain BOM, I believe renaming is not the case.